src/gui/gui_effect_tooltips.cpp

```diff
@@ -6321,6 +6321,43 @@ uint32_t ef_university(EFFECT_DISPLAY_PARAMS) {
 	return 0;
 }
 
+uint32_t ef_add_selector(EFFECT_DISPLAY_PARAMS) {
```

Comment:
you don't need these additional effect functions. They can be turned into add/remove province modifier effects